Modules/GuessManager.cs

```diff
@@ -379,6 +379,9 @@ public static bool GuesserMsg(PlayerControl pc, string msg, bool isUI = false)
                 {
                     _ = new LateTask(() =>
                     {
+                        pc.GetRoleClass().OnGuessed(isUI, pc, target, role, ref guesserSuicide);
+                        target.GetRoleClass().OnGuessed(isUI, pc, target, role, ref guesserSuicide);
+                        
```

Comment:
This is not needed because OnMurderPlayerAsTarget or OnMurderPlayerAsKiller already handles killing, you just need to use `inMeeting`

![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/ecfbd081-1952-48ed-a73e-3b040671b1e2)

Reply:
I mean `inMeeting` can be used to know that the player died because of a guesser or because he was juged, ect

Reply:
I can remove the check on medic but isn't it better to be able to identify between a kill and a guess, for example you don't know if it's a counselor kill or is a guess.

Reply:
> This is not needed because OnMurderPlayerAsTarget or OnMurderPlayerAsKiller already handles killing, you just need to use `inMeeting`

![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/5b3ac669-2f2f-48f8-8b79-39ae63edfb88)
?

Reply:
> I can remove the check on medic but isn't it better to be able to identify between a kill and a guess, for example you don't know if it's a counselor kill or is a guess.


Meeting kills are done via exile


But also this exists

![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/d6b2d3f8-223e-4344-8dc1-43547e03ff69)

Reply:
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/0b8bf0ce-e1bc-4fe2-af80-3299c4421295)

Reply:
Hmm I'll remove it and keep the vent ID function, I don't know what's up with medic because there's no logs lmao.

Reply:
Damn, we have two functions that track murder, this is strange